### ISSUE TITLE
Issue with stored_version in versioned carrierwave images on 1.8.0 & 1.8.1

### ISSUE THREAD

**Issue by ron** [cloudinary/cloudinary_gem#261]
I downgraded to 1.7.0 for now, because this issue doesn't seem to occur there.

I use the following image in my app, with a carrierwave upload:
image_tag(company.logo_url(:thumbnail)) 

In 1.8.1 it translates to:
http://res.cloudinary.com/salonized/image/upload/b_transparent,c_pad,g_center,h_150,w_150/V31Ai.png

In 1.7.0 it translates to: 
http://res.cloudinary.com/salonized/image/upload/b_transparent,c_pad,g_center,h_150,w_150/v1499335057/V31Ai.png

The stored_version (or version in the url) is empty and not used. This is only for versioned carrierwave images. The default image `company.logo_url` works correctly.

If I call it like this it also works correctly: `company.logo_url(:thumbnail, version: company.logo.stored_version)`

I did some debugging, and in 1.8.1 `self.stored_version` is nil on this line:
https://github.com/cloudinary/cloudinary_gem/blob/master/lib/cloudinary/carrier_wave.rb#L48

My Uploader looks like this:

```
class CompanyLogoUploader < CarrierWave::Uploader::Base
  include Cloudinary::CarrierWave

  process convert: 'png'
  process tags: ['company_logo']

  process resize_to_limit: [800, 800]

  version :thumbnail do
    resize_and_pad(150, 150)
  end

  def public_id
    model.public_id
  end

  def extension_white_list
    %w(jpg jpeg gif png)
  end
end
```

If you need any more info let me now.

**Comment by jughead** [comment]
Had probably the same issue. I had an uploaded image with cludinary like `v1501598359`. But I've got `v1` on CarrierWave versions (thumnails, etc.). I think we should add `:stored_version` here: https://github.com/cloudinary/cloudinary_gem/blob/1.8.1/lib/cloudinary/carrier_wave.rb#L15

**Comment by roeeba** [comment]
Hi @ron, @jughead. I apologise for the late reply. We're aware of this issue and will do our best to fix it as soon as we can. We'll update here once resolved.

**Comment by aleksandar-cloudinary** [comment]
I see this thread had remained open so I'm revisiting it. Thanks, @jughead for the PR. @ron - this should be fixed from versions 1.8.3+ of the gem, but if you still see any issues please re-open.
